### PR TITLE
docs: add feature flag section to targeting rule

### PR DESCRIPTION
**Pull request by kentakozuka**
Fixes https://github.com/bucketeer-io/bucketeer/issues/963

### CODE REVIEW

File: docs/feature-flags/creating-feature-flags/targeting.mdx [hunk 1]
```diff
@@ -73,6 +73,10 @@ The image below presents an example using two flags as prerequisites. Therefore,
   wSize="500px"
 />
 
+:::info Performance impact
+When defining rollout rules based on other feature flags, consider the performance impact on your system. If a flag to be evaluated has many dependencies, evaluating one flag may also evaluate many other flags, potentially affecting the performance of evaluation duration and client's network load.
```
Comment (cre8ivejp) [inline docs/feature-flags/creating-feature-flags/targeting.mdx:77]:
```suggestion
When defining rollout rules based on other feature flags, consider the impact of performance on your system. If a flag to be evaluated has many dependencies, evaluating one flag may also evaluate many other flags, potentially affecting the performance of evaluation duration and the client's network load.
```

File: docs/feature-flags/creating-feature-flags/targeting.mdx [hunk 2]
```diff
@@ -183,6 +187,22 @@ For example, consider you created a rollout rule using the **Date** condition fo
   wSize="500px"
 />
 
+### Feature flag
+
+You also can define rollout rules based on other feature flags. The **feature flag** option is similar to the [Prerequisites](/feature-flags/creating-feature-flags/targeting#prerequisites), meaning the condition is based on other feature flags' evaluations. The rule's condition is satisfied if the user is evaluated with the expected variation on the feature flag.
+
+For example, consider that you created a rollout rule using the **Feature flag** condition based on the `true` variation of the `foo feature flag.
```
Comment (cre8ivejp) [inline docs/feature-flags/creating-feature-flags/targeting.mdx:194]:
```suggestion
For example, consider that you created a rollout rule using the **Feature flag** condition based on the `true` variation of the foo feature flag.
```

File: docs/feature-flags/creating-feature-flags/targeting.mdx [hunk 3]
```diff
@@ -183,6 +187,22 @@ For example, consider you created a rollout rule using the **Date** condition fo
   wSize="500px"
 />
 
+### Feature flag
+
+You also can define rollout rules based on other feature flags. The **feature flag** option is similar to the [Prerequisites](/feature-flags/creating-feature-flags/targeting#prerequisites), meaning the condition is based on other feature flags' evaluations. The rule's condition is satisfied if the user is evaluated with the expected variation on the feature flag.
```
Comment (cre8ivejp) [inline docs/feature-flags/creating-feature-flags/targeting.mdx:192]:
```suggestion
You also can define rollout rules based on other feature flags. The **Feature flag** option is similar to the [Prerequisites](/feature-flags/creating-feature-flags/targeting#prerequisites), meaning the condition is based on other feature flags' evaluations. The rule's condition is satisfied if the user is evaluated with the expected variation on the feature flag.
```

**Comment by kentakozuka** [comment]
@cre8ivejp fixed 👍

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**